src/ray/common/ray_object.cc

```diff
@@ -125,9 +131,10 @@ bool RayObject::IsInPlasmaError() const {
   if (metadata_ == nullptr) {
     return false;
   }
-  const std::string metadata(reinterpret_cast<const char *>(metadata_->Data()),
-                             metadata_->Size());
-  return metadata == std::to_string(ray::rpc::ErrorType::OBJECT_IN_PLASMA);
+  const std::string_view metadata(reinterpret_cast<const char *>(metadata_->Data()),
+                                  metadata_->Size());
+  // Keep in sync with common.proto.
+  return metadata == "4"; /*=std::to_string(ray::rpc::ErrorType::OBJECT_IN_PLASMA)*/
```

Comment:
Maybe we can add a line here to be fool proof

  static_assert(ray::rpc::ErrorType::OBJECT_IN_PLASMA == 4);